lib/src/generators/openapi2/openapi2-specification.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
There are not too many `any`s, maybe it'd be better to have per-line disabling to prevent more from sneaking in?